open_instruct/dataset_transformation.py

```diff
@@ -1265,8 +1311,8 @@ def rlvr_filter_v1(
     "preference_filter_v1": (preference_filter_v1, "filter"),
     "preference_tulu_tokenize_and_truncate_v1": (preference_tulu_tokenize_and_truncate_v1_2, "map"),
     "preference_tulu_filter_v1": (preference_tulu_filter_v1, "filter"),
-    "rlvr_tokenize_v1": (rlvr_tokenize_v2, "map"),
-    "rlvr_filter_v1": (rlvr_filter_v1, "filter"),
+    "rlvr_tokenize_v1": (rlvr_tokenize_v3, "map"),
```

Comment:
Why are these different versions? (why not both v1 or v3?)

Reply:
This is an old design Costa felt strongly about --that maybe you could have multiple tokenize and filter functions and give these diff names, and so swap prepro easily at script calling time... but then the default options in the script use v1, so we ended up just updating this mapping so that people could get new prepro without updating scripts.

Reply:
The _v3 name for the function itself helps avoid the HF cache invalidation issue: now its a new function with a new name, so it won't try to reuse the old caches.